homeassistant/components/sensor/version.py

```diff
@@ -33,20 +33,21 @@
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_BETA, default=False): cv.boolean,
-    vol.Optional(CONF_IMAGE, default=DEFAULT_IMAGE): cv.string,
+    vol.Optional(CONF_IMAGE, default=DEFAULT_IMAGE): vol.All(vol.Lower,
```

Comment:
Invert `vol.Lower` and `cv.string` order, to make sure it's a string before lower casing it. `vol.All` performs validations in sequence.